wordpress-mobile/WordPress-FluxC-Android
# Allow `ReleaseStack_WCBase` tests to specify test site

Closes #2091 

This draft is ready to review. I mark it as draft as I didn't add correct values to `test.properties` as I'm not sure if this is direction we would like to resolve this issue.

### Description

This PR allows to select test site for every site depending on `ReleaseStack_WCBase`, but preserving old, default behaviour of picking first available site that was before.

### How to test?

Please update site id declared in `ReleaseStack_WCPayTest#22` with correct site id (please DM me for it if you don't have it by hand) and run:
- `ReleaseStack_WCPayTest`
- Any other `ReleaseStack_WC*` test (if it fails, make sure it doesn't on `develop`)
